client/src/modules/stock/movements/modals/search.modal.html

```diff
@@ -12,80 +12,100 @@
   </div>
 
   <div class="modal-body">
+    <uib-tabset>
+      <uib-tab index="0" heading="{{'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
+        <div class="tab-body">
+          <!-- movements  -->
+          <div class="form-group">
+            <fieldset>
+              <legend translate>STOCK.MOVEMENTS</legend>
```

Comment:
I know you didn't write this code originally, but it would be really nice to make this use bootstrap's [checkbox and radio button style](https://getbootstrap.com/docs/3.3/css/#checkboxes-and-radios).  It should look something like the exampel below:
```html
<div class="radio">
  <label>
     <input type="radio" name="is_exit" value="0" ng-model="$ctrl.searchQueries.is_exit"> 
     <span translate>STOCK.INPUT</span>
  </label>
</div>
<div class="radio">
  <label>
    <input type="radio" name="is_exit" value="0" ng-model="$ctrl.searchQueries.is_exit"> 
    <span translate>STOCK.OUTPUT</span>
  </label>
</div>
```
This has the following advantages:
 1. It will look like most other inputs in the application
 2. No need for the `&nbsp;` extra tags.